### PR TITLE
add makefile 

### PR DESCRIPTION
**Pull request by MohanChhabaria**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+BUILD_NAME=cdc-portal
+BUILD_TAG=$$(git log -1 --pretty=%h)
+
+build:
+	@docker build -t ${BUILD_NAME}-backend:${BUILD_TAG} -t ${BUILD_NAME}-backend:latest -f Dockerfile .
```
Comment (DarshitJain04) [inline Makefile:5]: @MohanChhabaria the dockerfile for Backend is jnr ./backend directory
You'll have to change it accordingly

Comment (MohanChhabaria) [reply Makefile]: Thanks @DarshitJain04  for pointing this out.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,21 @@
+BUILD_NAME=cdc-portal
+BUILD_TAG=$$(git log -1 --pretty=%h)
+
+build:
+	@docker build -t ${BUILD_NAME}-backend:${BUILD_TAG} -t ${BUILD_NAME}-backend:latest -f backend/Dockerfile backend.
```
Comment (Tanuj22) [inline Makefile:5]: why `backend.`. It should be just `backend` right?

Comment (MohanChhabaria) [reply Makefile]: Yeah, changes done

**Review by Tanuj22 — APPROVED** [review]
LTGM!

**Pull request merged**